servers/rendering/renderer_viewport.cpp

```diff
@@ -764,6 +763,7 @@ void RendererViewport::draw_viewports(bool p_swap_buffers) {
 							blit_to_screen_list[vp->viewport_to_screen].push_back(blits[b]);
 						}
 					}
+					RSG::rasterizer->end_viewport(p_swap_buffers && blits.size() > 0);
```

Comment:
```suggestion
					RSG::rasterizer->end_viewport(p_swap_buffers && !blits.is_empty());
```
Optional style suggestion